src/app/(main)/layout.tsx

```diff
@@ -0,0 +1,20 @@
+import Navbar from "@/components/common/Navbar/Navbar";
+import type { Metadata } from "next";
+
+export const metadata: Metadata = {
```

Comment:
u dont need to define Metadata type here

Reply:
Yes we do, takay we can have separate layouts for things and their specified metadata aswell... or is there a better way of approaching this in nextjs? Since according to next's documentation this is the favoured method